Makefile

```diff
@@ -7,11 +7,16 @@ setup-dev: requirements.txt requirements_test.txt
 unittest:
 	pytest -n 2 tests/
 
+unittest-not-slow:
+	pytest -k 'not slow' -n 2 --durations=5 tests/
+
 lint:
 	pytest --pylint -m pylint --pylint-jobs=2 .
 
 # Run unittests then linting
 test: unittest lint
 
+test-fast: lint unittest-not-slow
```

Comment:
fwiw linting isn't super quick - maybe unittests should run first?

Reply:
For me linting takes about as long as 'not slow' test the first time and much faster when run a second time.